bower.json

```diff
@@ -11,17 +11,16 @@
     "output"
   ],
   "dependencies": {
-    "purescript-string-parsers": "^5.0",
-    "purescript-aff": "^5.1",
-    "purescript-transformers": "^4.2",
-    "purescript-generics-rep": "^6.1"
+    "purescript-string-parsers": "^6.0.0",
+    "purescript-aff": "^6.0.0",
+    "purescript-transformers": "^5.0.0"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/cdepillabout/purescript-email-validate"
```

Comment:
Oh! I forgot to mention, we need to update this to be the same url as the registry for publishing to pursuit. In almost every case that just means changing this to https:// instead of git://